src/modules/CurrencyModule.cpp

```diff
@@ -985,11 +991,8 @@ void CurrencyModule::OnSave(QJsonDocument& doc) const
 	{
 		QJsonObject serverJSON;
 		
-		for (auto user = guildList[server].begin(); user != guildList[server].end(); user++) {
-			if (m_client->GetName(server, user->userId) == "")
-			{
-				continue;
-			}
```

Comment:
Why do I get a feeling that this was the bad guy all along..

Reply:
It's what *does* the bad job, but it isn't what causes the issue...